### PR TITLE
Made headline structure like FastAPI docs and added screenshot

### PR DESCRIPTION
**Pull request by antonpirker**
Just some smaller changes.

### CODE REVIEW

File: src/platforms/python/guides/pymongo/index.mdx [hunk 1]
```diff
@@ -44,11 +44,21 @@ automatically.
 
 </Note>
 
-## Other MongoDB libraries
+## Behavior
+
+The following information about your MonogDB queries will be available to you on Sentry.io:
+
+- Performance traces for all MongoDB queries
+- Breadcrumbs for all MongoDB queryies
```
Comment (Agalin) [inline src/platforms/python/guides/pymongo/index.mdx]:
```suggestion
- Breadcrumbs for all MongoDB queries
```

File: src/platforms/python/guides/pymongo/index.mdx [hunk 2]
```diff
@@ -44,11 +44,21 @@ automatically.
...
+- Personal identifiable information (PII) will be stripped from all MongoDB queries. (This was tested for PyMongo 4.2 and below, but "should" also be future proof)
```
Comment (Agalin) [inline src/platforms/python/guides/pymongo/index.mdx]:
```suggestion
- Personal identifiable information (PII) will be stripped from all MongoDB queries if `send_default_pii` is disabled in the SDK. (This was tested for PyMongo 4.2 and below, but "should" also be future proof)
```

Or something similar. If I was reading the current version without knowing the implementation I'd expect PII to never be sent.

File: src/platforms/python/guides/pymongo/index.mdx [hunk 3]
```diff
@@ -44,11 +44,21 @@ automatically.
...
+
+![MongoDB query details are shown as waterfall diagram](mongodb-query-performance-details.png)
```
Comment (Agalin) [inline src/platforms/python/guides/pymongo/index.mdx]:
```suggestion
![MongoDB query details are shown as a waterfall diagram](mongodb-query-performance-details.png)
```

**Comment by antonpirker** [comment]
Ok, updated the docs

**Comment by Agalin** [comment]
Hmm. PII discussion got resolved by accident. Can you also check this one?

**Comment by antonpirker** [comment]
done

**Pull request merged**